mern-ecommerce/controllers/auth.js

```diff
@@ -72,11 +72,13 @@ exports.isAuth = (req, res, next) => {
   next();
 };
 
-exports.isAdmin = (req, res, next) => {
-  if (req.profile.role === 0) {
-    return res.status(403).json({
-      error: 'Admin resource! Access denied',
+exports.isAdmiaan = (req, res, next) => {
```

Comment:
"Typo in 'rola', should be 'role'. Also, consider using a more descriptive error message and a consistent status code (403) for access denied errors."